web/src/lib/utils/file-uploader.ts

```diff
@@ -97,7 +93,12 @@ async function fileUploader(
 		formData.append('deviceId', 'WEB');
 
 		// Get asset type
-		formData.append('assetType', assetType);
+		if (fileExtension.toLowerCase() == 'raf' || fileExtension.toLowerCase() == 'srw') {
```

Comment:
Maybe a helper method here to check for raw file types?

Reply:
Did that.